dist/react-filterbar.js

```diff
@@ -11135,9 +11146,40 @@ var FilterBarStore = exports.FilterBarStore = (function () {
     updateFilter: {
       value: function updateFilter(filterUid, propKey, propValue) {
         this.filters[filterUid][propKey] = propValue;
+        if (propKey === "value") this.deactivateQuickFiltersBasedOnFilterValue(filterUid, propValue, this.activeQuickFilters());
         this.emitChange();
       }
     },
+    deactivateQuickFiltersBasedOnFilterValue: {
+      value: function deactivateQuickFiltersBasedOnFilterValue(filterName, filterValue, quickFilters) {
+        quickFilters.map(function (quickFilter) {
+          Object.keys(quickFilter.filters).map(function (quickFilterName) {
+            var quickFilterFilter = quickFilter.filters[quickFilterName];
+            if (quickFilterFilter.filter === filterName) {
+              if (typeof quickFilterFilter.value === "object") {
```

Comment:
@louman I dont think we have any quickfilters that are arrays of ids .. but the multiselect filter values are arrays
will they work as well ? ie can we slot in a quickfilter that is an array of user_ids like MyTeam